### ISSUE TITLE
Accessibility audit of current CMS design system

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9993]
## Background

To ensure that the Sitewide CMS team can support building accessible products across the program, the [CMS editorial design system](https://www.figma.com/file/ScABI31f3MRFEvQ8WitVPa/CMS-Design-System?node-id=0%3A1) should be audited for opportunities. 

## Acceptance Criteria

- [x] Accessibility lead audits the Figma files of the CMS design system
- [x] Accessibility concerns are documented 
- [x] Design opportunities are shared with Sitewide CMS interaction designer

**Comment by laflannery** [comment]
After looking at the current Design system I have created a few tickets for issues that I believe could be completed/looked at now as they are all small, simple issues that wouldn't necessarily be dependent on other elements:

- #10070 
- #10071 
- #10072 

Overall, I think in order to help facilitate creating accessible products within the CMS using the Design System we may want to look at streamlining what we currently have, possibly creating one single source of truth.

One possible idea on how best to do this might be to start with how Cindy and I already started with the links - we determined a solid link style (colors, fonts, states, etc.)  All of this should be taken and put into a new document (Style Guide?) and then we can build on it from there.  We define buttons - then add it to the Style Guide.  Then we define Headings - add it to the Guide.  And so on.... That's just my thought and I'm open for discussion!

**Comment by EWashb** [comment]
I like these next steps! I think this is done and we can work off the new tickets.
